skimage/measure/_regionprops.py

```diff
@@ -601,7 +603,23 @@ def perimeter(image, neighbourhood=4):
     .. [1] K. Benkrid, D. Crookes. Design and FPGA Implementation of
            a Perimeter Estimator. The Queen's University of Belfast.
            http://www.cs.qub.ac.uk/~d.crookes/webpubs/papers/perimeter.doc
+
+    Examples
+    --------
+    >>> from skimage import data, util
+    >>> from skimage.measure import label
+    >>> # coins image (binary)
+    >>> img_coins = util.img_as_ubyte(data.coins()) > 110
```

Comment:
```suggestion
     >>> img_coins = data.coins() > 110
```

I don't think the conversion is necessary here.

Reply:
Yes it works !